engine/image-loader/image-loader.lisp

```diff
@@ -0,0 +1,37 @@
+(defpackage lbge.image-loader
+  (:use :cl)
+  (:export load-image make-image))
+(in-package lbge.image-loader)
+
+(defclass image ()
+  ((width :reader  width
+          :initarg :width
+          :initform nil
```

Comment:
Note for the future: maybe remove these nils, they aren't valid values for integer and the image isn't valid if any of them are nil.